modules/claims_api/lib/claims_api/claim_logger.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  # Logger for ClaimsApi, mostly used to format a print statement
+  # Uses the standard Rails logger
+  #
+  #   ClaimsApi::Logger.log('526ez', claim_id: 'id_here')
+  #   # => "ClaimsApi :: 526ez :: Claim ID: id_here"
+  class Logger
+    LEVELS = %i[debug info warn error fatal unknown].freeze
+
+    def self.log(tag, **params)
+      level = pick_level(**params)
+      params.delete(:level)
+      msg = format_msg(tag, **params)
+
+      Rails.logger.send(level, msg)
+      msg
+    end
+
+    def self.pick_level(**params)
+      params.key?(:level) && params[:level].to_sym.in?(LEVELS) ? params[:level].to_sym : :info
+    end
+
+    def self.format_msg(tag, **params)
+      msg = ['ClaimsApi', tag]
+      case tag
+      when '526ez'
+        msg.append("Claim ID: #{params[:claim_id]}") if params[:claim_id].present?
+        msg.append("VMBS ID: #{params[:vbms_id]}") if params[:vbms_id].present?
+        msg.append("autoCrestPDFGenerationDisabled: #{params[:pdf_gen_dis]}") || false
```

Comment:
Two things here.

1. It's `autoCestPDFGenerationDisabled` (not `autoCrestPDFGenerationDisabled`)
2. Isn't this always going to append `autoCrestPDFGenerationDisabled` to the log message?  Like, even when we don't care about logging the state of that attribute?